Documentation/gettingstarted/minikube_intro.rst

```diff
@@ -19,7 +19,7 @@ dependency.
 
 ::
 
-    $ minikube start --network-plugin=cni --bootstrapper=localkube --memory=4096 --extra-config=apiserver.Authorization.Mode=RBAC
+    $ minikube start --network-plugin=cni --bootstrapper=localkube --memory=5120 --extra-config=apiserver.Authorization.Mode=RBAC
```

Comment:
5GB... this is getting a little out of hand...

Reply:
I know. But with only 4GB the VM was running out of swap and ground to a halt.